charts/bookkeeper-operator/values.yaml

```diff
@@ -3,8 +3,8 @@
 # Declare variables to be passed into your templates.
 
 image:
-  repository: pravega/pravega-operator
-  tag: 0.4.0
+  repository: srishtithakkar/bookkeeper-operator
```

Comment:
Its fine to use your private repo for testing, but here we should have "pravega/bookkeeper-operator" and tag should be "latest"

Reply:
done